magento/inventory
# MSI-2677: Timestamp of customer notification (when Ready for Pickup was clicked) should be displayed in order details.

### Description (*)


### Fixed Issues (if relevant)

1. https://github.com/magento/inventory/issues/2677: Timestamp of customer notification (when Ready for Pickup was clicked) should be displayed in order details

### Manual testing scenarios (*)

1. Create order with store pickup delivery method.
2. Press 'Notify Order for Pick-Up' on admin order page.
3. Verify, comment has been added to order with timestamp


### Questions or comments


### Contribution checklist (*)
